app/lib/generator/certificate.js

```diff
@@ -0,0 +1,89 @@
+const PDFDocument = require('pdfkit')
+
+const { findFont } = require('./fonts')
+const { formatDate } = require('../date-helpers')
+
+const processTemplate = (doc, files, values) => {
```

Comment:
I found the param name 'files' a bit confusing here. Yes, it's 3 files but is really the template config (template json + logo file + signature file). I had to walk through the code to find out why you were passing a series of files here.